automated install/uninstall.sh

```diff
@@ -131,17 +113,8 @@ removeNoPurge() {
         echo -e "  ${TICK} Removed /etc/cron.d/pihole"
     fi
 
-    if package_check lighttpd > /dev/null; then
-        ${SUDO} rm -rf /etc/lighttpd/ &> /dev/null
-        echo -e "  ${TICK} Removed lighttpd"
-    else
-        if [ -f /etc/lighttpd/lighttpd.conf.orig ]; then
-            ${SUDO} mv /etc/lighttpd/lighttpd.conf.orig /etc/lighttpd/lighttpd.conf
-        fi
-    fi
-
     ${SUDO} rm -f /etc/dnsmasq.d/adList.conf &> /dev/null
-    ${SUDO} rm -f /etc/dnsmasq.d/01-pihole.conf &> /dev/null
+    ${SUDO} rm -f /etc/dnsmasq.d/pihole.conf &> /dev/null
```

Comment:
Should this file, despite being generated by the API, not still be named `01-pihole.conf` to ensure it is read in first? I thought files in `/etc/dnsmasq.` were read in alphanumerical order?

Reply:
The reason for stripping the prefix was that there is only one Pi-hole config file, and the prefix is unnecessary. The order of the config files does not matter much, I don't think dnsmasq really cares. If there is a conflicting option, it would throw an error no matter the order.

Reply:
i know when you specify multiple dns entries to be distributed it recommended to them in a file that is read as close to last as possible to ensure that they are used. 

https://www.reddit.com/r/pihole/comments/b88ciu/question_im_using_2_piholes_how_to_make_one_hand/

Reply:
@mwoolweaver I do not see that sentiment in the link you shared. Can you link to the comment with this information?

> close to last as possible to ensure that they are used.

If the config file is read, it will be used. Moving its position in the order of config files does not change if it will be parsed and used.

Reply:
https://github.com/pi-hole/pi-hole/issues/975#issuecomment-281074624

https://discourse.pi-hole.net/t/pihole-update-how-to-retain-my-dnsmaq-settings-dhcp-interaface-etc/269/13?u=mwoolweaver1

https://discourse.pi-hole.net/t/allow-more-granular-control-of-dhcp-lease-time/16500/11?u=mwoolweaver1

https://discourse.pi-hole.net/t/dhcp-option-66-tftp/8437/5?u=mwoolweaver1

https://discourse.pi-hole.net/t/ignore-explicit-whitelisted-domains-from-query-log/2335/14?u=mwoolweaver1


It does seem that everyone’s opinion isn’t the same.